### PR TITLE
URL's have changed

### PR DESCRIPTION
**Pull request by mossyhub**
there are now multiple URLs for  sending commands and the commands have changed. been building and testing this for a few days now. seems solid. cant get the poll (remote status) stuff to work though for some reason. Using Fiddler on mopar.com to build out the poll URL and params...it seems i have it all right, it does a GET on the same URL as the POST... but I gave up for now.

documentation for Home Assist for the mopar sensor needs changed for this as well.

### CODE REVIEW

File: motorparts/__init__.py [hunk 1]
```diff
@@ -27,18 +27,20 @@
 TOKEN_URL = 'https://www.mopar.com/moparsvc/token'
 TOW_URL = 'https://www.mopar.com/moparsvc/vehicle/tow-guide/vin'
 VHR_URL = 'https://www.mopar.com/moparsvc/getVHR'
-REMOTE_COMMAND_URL = 'https://www.mopar.com/moparsvc/remoteCommands'
-REMOTE_STATUS_URL = 'https://www.mopar.com/moparsvc/vehicle/remote/status'
+REMOTE_LOCK_COMMAND_URL = 'https://www.mopar.com/moparsvc/connect/lock'
+REMOTE_ENGINE_COMMAND_URL = 'https://www.mopar.com/moparsvc/connect/engine'
+REMOTE_ALARM_COMMAND_URL = 'https://www.mopar.com/moparsvc/connect/alarm'
+global REMOTE_COMMAND_URL
```
Comment (happyleavesaoc) [inline motorparts/__init__.py]: Any design should not have to use `global`. Modify `remote_command(...)` to check the `command` arg to select the correct const url, that's probably the easiest thing to do.

**Review by happyleavesaoc — CHANGES_REQUESTED** [review]
Run `tox` get code style notes. Also left one comment about how to handle the varying URLs.

File: motorparts/__init__.py (hunk 1 above)
Comment (mossyhub) [reply motorparts/__init__.py]: I think I have it fixed now. apologize for any errors or style concerns. this is my first time using Python.

File: motorparts/__init__.py [hunk 2]
```diff
@@ -30,8 +30,6 @@
 REMOTE_LOCK_COMMAND_URL = 'https://www.mopar.com/moparsvc/connect/lock'
 REMOTE_ENGINE_COMMAND_URL = 'https://www.mopar.com/moparsvc/connect/engine'
 REMOTE_ALARM_COMMAND_URL = 'https://www.mopar.com/moparsvc/connect/alarm'
-global REMOTE_COMMAND_URL
```
Comment (mossyhub) [inline motorparts/__init__.py]: removed. this was only global for the sake of passing it to _remote_status

**Comment by mossyhub** [comment]
Anything I can do to get this merged.?I think its all good according to your comments before. I would really love to use this in Home Assistant but dont want to step on your toes and  release separately.

**Comment by happyleavesaoc** [comment]
Hi. I merged this and then pushed another commit to update some stylistic stuff (note, you can run the style checker by `pip install tox` and then running `tox`).

Please give it a test, if it all still works, I'll publish the new version to pypi.

**Comment by mossyhub** [comment]
works great. i had never touched python before this so honestly was just copy/pasting from your code to NotePad++ to the python IDLE as i made changes....just barely today looked into an IDE and how Tox works. Please update the mopar.py on Home Assist to match your version when you publish and this will be perfect. "OK Google...start my family wagon!"

**Comment by rohankapoorcom** [comment]
@happyleavesaoc It looks like this was never published on pypi. Would you be able to get it published so that we can update and fix Home Assistant? See: https://github.com/home-assistant/home-assistant/issues/16379

**Comment by happyleavesaoc** [comment]
@rohankapoorcom `1.1.0` pushed to pypi.

**Comment by rohankapoorcom** [comment]
@happyleavesaoc upgrading Home Assistant to use 1.1.0 in https://github.com/home-assistant/home-assistant/pull/21602

**Pull request merged**